### ISSUE TITLE
Asking for stanford dog training hyperparameters

### ISSUE THREAD

**Issue by M1andy** [ivclab/CVS#1]
I'm doing the fine-grained dataset training, and I have already finished the dog and jd-product these two datasets. The method works on the jd-product dataset, while in the dog dataset there is a considerable gap between my run and the papers.

my current config is basiclly the same with the default settings, only a few has changed.

in `session 0`, the `epochs` is `50`
in `other sessions`, the `epochs` is `30`
and `batch size=256` and `learning rate=1e-3` remain the same in all sessions

and the result image is down below.
<img width="991" alt="Snipaste_2023-06-08_14-57-24" src="https://github.com/ivclab/CVS/assets/54304769/f0188f7f-bcc4-4b26-8f89-83c631735776">

**Comment by bigchou** [comment]
Sorry for the late reply. 
My training hyperparameters are as follows (across all sessions):

--n_epochs 100 --bs 64 --sched_name none --arch resnet50 --pretrain --dataset dog --lr 1e-4 --buffer 600 --loss_m --loss_d --alpha 10.0 --replay

**Comment by M1andy** [comment]
Thank you for the reply.
